README.md

```diff
@@ -1,53 +1,41 @@
-# AnyCloud
-
-AnyCloud is a Lambda alternative for Node.js that works with multiple cloud providers. We’re building the features of Lambda while fixing the problems we have run into while using it. Namely:
+<div align="center">
+  <img src="./assets/anycloud.png" alt="drawing" width="180"/>
```

Comment:
I can't comment on images. Did you try out the other way around? The cloud containing the lines and no circle?

Reply:
It was giving me a really hard time to do it properly and I didn't want to spend too long on it. I also wasn't liking the look of it.